google/cloud/bigtable/examples/bigtable_table_admin_backup_snippets.cc

```diff
@@ -166,6 +166,59 @@ void RestoreTableFromInstance(google::cloud::bigtable::TableAdmin const& admin,
   (admin, argv.at(0), argv.at(1), argv.at(2), argv.at(3));
 }
 
+void GetIamPolicy(google::cloud::bigtable::TableAdmin const& admin,
+                  std::vector<std::string> const& argv) {
+  //! [get backup iam policy]
+  namespace cbt = google::cloud::bigtable;
+  using google::cloud::StatusOr;
+  [](cbt::TableAdmin admin, std::string const& cluster_id,
+     std::string const& backup_id) {
+    StatusOr<google::iam::v1::Policy> policy =
+        admin.GetIamPolicy(cluster_id, backup_id);
+    if (!policy) throw std::runtime_error(policy.status().message());
+    std::cout << "The IAM Policy for "
+              << admin.BackupName(cluster_id, backup_id) << " is\n"
```

Comment:
would just `backup_id` be preferred over the full name?

Reply:
And be wary that `DebugString()` already contains a trailing newline.

Reply:
I left in the extra newline at the end just for consistency. The other examples in the file look like `... << proto->DebugString() << "\n"`